README.md

```diff
@@ -135,11 +138,14 @@ Flags:
 * Either `ARTI_USERNAME` and `ARTI_PASSWORD` or `ARTI_ACCESS_TOKEN` environment variables has to be set.
 
 ### Metrics
+The quota metrics are only available if the feature to collect them is enabled.
 
 Some metrics are not available with Artifactory OSS license. The exporter returns the following metrics:
 
 | Metric | Description | Labels | OSS support |
 | ------ | ----------- | ------ | ------ |
+| artifactory_storage_repo_quota_used_percent | Percentage of quota used |`name`, `package_type`, `type` | ? Need to confirm it works with OSS  |
```

Comment:
I would recommend using this format:
`TODO: confirm it works with OSS`

Reply:
changed line 147